.gitignore

```diff
@@ -0,0 +1,8 @@
+.sublime-workspace
+/coverage
+/docs
+/node_modules
+/npm-debug.log
+/tmp
+.npmrc-temp
+npmrc-temp
```

Comment:
(possible future) workarounds for docker, and handling of registry secrets in `npmrc`